src/main/resources/component.mustache

```diff
@@ -14,9 +14,9 @@ We will support this new release until *{{parent.schedule.formattedEOLDate}}* an
 support statement from your team for this Spring Boot version in the form of the following information:
 - Product name and version (e.g. JWS - 5.3.1)
 - Date of end of support
-- Supported artifact GAV (if changed) or simply version (if name hasn't changed) (e.g. org.apache.tomcat.embed:tomcat-embed-core:9.0.30-redhat-00001)
-- PNC/Indy URL (e.g. [http://indy.psi.redhat.com/browse/maven/]...)
-- MRRC URL, if already released (e.g. [https://maven.repository.redhat.com/ga/]...)
+- Supported artifact GAV (if changed) or simply (productized) version (if name hasn't changed) (e.g. org.apache.tomcat.embed:tomcat-embed-core:9.0.30-redhat-00001)
```

Comment:
I think that we should rephrased again this sentence as the text is certainly still too complex to be understood clearly ->  `Supported artefact GAV (if changed) or simply (productized) version (if name hasn't changed)`.
That text should explain that we expect `either a new productized version or an unchanged version which has been tested against our SB version ...`

Reply:
From my point of view the @metacosm sounds clear to me. 
Another way to say that can be: 
- Current Supported Artifact (GAV) either didn't change or got refreshed